tools/tertiary-analysis/atlas-data-import/atlas_import_sdrf_files.xml

```diff
@@ -0,0 +1,33 @@
+<tool id="atlas_import_sdrf_files" name="Atlas import: get sdrf files" version="@TOOL_VERSION@+galaxy0"  profile="@PROFILE@">
```

Comment:
I don't think that is worth having a whole tool just for this, I would add it to the main downloader with an optional activation switch.

Reply:
@pcm32 I was thinking so too initially, but then decided to make a separate tool, because it's a different use case. In atlas-data-import, we specify single accession and import data. Here, the tool finds available classifiers for _all_ datasets, unless specified otherwise, and imports them. Putting all this into one script will be a mess.